docs/sources/source/nlp/megatron_finetuning.rst

```diff
@@ -0,0 +1,36 @@
+Megatron-LM for Downstream Tasks
+================================
+
+Megatron :cite:`nlp-megatron-lm-shoeybi2020megatron` is a large, powerful transformer developed by the Applied Deep Learning Research team at NVIDIA.
+More details could be found in `Megatron-LM github repo <https://github.com/NVIDIA/Megatron-LM>`_.
+
+In order to finetune a pretrained Megatron BERT language model on NLP downstream tasks from `examples/nlp  <https://github.com/NVIDIA/NeMo/tree/master/examples/nlp>`_, specify the pretrained_model_name like this: 
+
+.. code-block:: bash
+
+    --pretrained_model_name megatron-bert-345m-uncased
+
+For example, to finetune SQuAD v1.1 with Megatron-LM, run:
+
+.. code-block:: bash
+
+    python question_answering_squad.py  \
+    --train_file PATH_TO_DATA_DIR/squad/v1.1/train-v1.1.json  \
+    --eval_file PATH_TO_DATA_DIR/squad/v1.1/dev-v1.1.json \
+    --pretrained_model_name megatron-bert-345m-uncased
```

Comment:
whats the difference between megatron-bert-345m-uncased and megatron-bert-uncased? why do we need to have a special case for megatron-bert-345m-uncased